api/v1/submissions/PKPSubmissionController.php

```diff
@@ -612,6 +612,11 @@ public function edit(Request $illuminateRequest): JsonResponse
 
         $params = $this->convertStringsToSchema(PKPSchemaService::SCHEMA_SUBMISSION, $illuminateRequest->input());
 
+        // Submission language can not be changed when there are more than one publication or a publication's status is published
+        if (count($submission->getData('publications')) > 1 || $submission->getLatestPublication()->getData('status') === PKPSubmission::STATUS_PUBLISHED) {
```

Comment:
Hmmm... This edit can be called also for other changes, not only the submission language, so this condition should not be applied in general, I think

Reply:
Yes, forgot to add `if (isset($params['locale']) && ...`